<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditLayer</class>
 <widget class="QWidget" name="EditLayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit layer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#EditLayer {
	background-color: rgb(40, 40, 40);
}
QLabel {
	color: rgb(225, 225, 225);
	font: 16px &quot;Verdana&quot;;
}
QCheckBox {
	color: rgb(225, 225, 225);
	background-color: rgb(40, 40, 40);
	font: bold 11px &quot;Tahoma&quot;;
}
QPushButton, QLineEdit, QSpinBox {
	border: none;
	background-color: rgb(30, 30, 30);
	color: rgb(200, 200, 200);
}
QPushButton:hover, QLineEdit:focus, QSpinBox:focus {
	border-color: rgb(0, 128, 255);
	color: rgb(255, 255, 255);
}
QPushButton:hover:pressed {
	background-color: rgb(25, 25, 25);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb_layername">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>34</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>21</height>
    </size>
   </property>
   <property name="placeholderText">
    <string>New layer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 128, 255);
color: rgb(225, 225, 225);</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="cb_hidden">
      <property name="toolTip">
       <string>Only you can edit this layer</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Hidden</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cb_private">
      <property name="toolTip">
       <string>Only you can edit this layer</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Private</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
